packages/react/src/I18nProvider.tsx

```diff
@@ -12,15 +12,13 @@ export type I18nProviderProps = Omit<I18nContext, "_"> & {
   children?: React.ReactNode
 }
 
-export const LinguiContext = React.createContext<I18nContext>(null)
+export const LinguiContext = React.createContext<I18nContext | null>(null)
 
 export function useLingui(): I18nContext {
-  const context = React.useContext<I18nContext>(LinguiContext)
+  const context = React.useContext(LinguiContext)
 
-  if (process.env.NODE_ENV !== "production") {
```

Comment:
small change: I remove the condition which is a change can influence runtime behavior, but I think it's for the better.

before, if you ran into situation that you use useLingui without I18nProvider, you'd get something like "cannot read property i18n of null`. Eventually you'd figure out what the cause is, but now you'll see it directly in the error message so the investigation is easier.

For the app user, it'd be a crash (perhaps caught by an error boundary) either way.

Reply:
I think it was done to save some bytes in production builds. While you're developing, you see all useful error messages.